modAionImpl/test/org/aion/zero/impl/sync/SyncStatsTest.java

```diff
@@ -0,0 +1,176 @@
+package org.aion.zero.impl.sync;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.aion.zero.impl.BlockchainTestUtils.generateAccounts;
+import static org.aion.zero.impl.BlockchainTestUtils.generateNewBlock;
+import static org.aion.zero.impl.BlockchainTestUtils.generateRandomChain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.aion.crypto.ECKey;
+import org.aion.mcf.core.ImportResult;
+import org.aion.zero.impl.StandaloneBlockchain;
+import org.aion.zero.impl.types.AionBlock;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SyncStatsTest {
```

Comment:
please add tests for behavior when the stats are requested but there is no data gathered yet, to ensure that there are no exceptions thrown

Reply:
New tests where added to cover the cases when no statistics have been yet collected